plugin/trino-hive/src/main/java/io/trino/plugin/hive/PropertiesSystemTableProvider.java

```diff
@@ -60,6 +64,9 @@
         if (table.isEmpty()) {
             throw new TableNotFoundException(tableName);
         }
+        if (isDeltaLakeTable(table.get())) {
+            throw new TrinoException(HIVE_UNSUPPORTED_FORMAT, format("Cannot query Delta Lake table '%s'", sourceTableName));
```

Comment:
Same for `$partitions`

Reply:
This is marked resolved. what is the resolution? I am not seeing changes in `PartitionsSystemTableProvider`, but i guess i am missing something.

Reply:
I was planning to implement the functionality in the `PartitionsSystemTableProvider` , but dropped the idea after seeing the call to `io.trino.plugin.hive.HiveMetadata#getTableHandle` method, which has  already the check for delta lake format.

See 
https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java#L433-L435

and do note that the test has been updated to check the query of `$partitions` system table as well